PythonRpcServer/server.py

```diff
@@ -62,13 +64,18 @@ def GetYoutubePlaylistRPC(self, request, context):
             return ct_pb2.JsonString()
     
     def DownloadYoutubeVideoRPC(self, request, context):
+        youtubeprovider = YoutubeProvider()
         filePath, ext = youtubeprovider.getMedia(request)
         return ct_pb2.File(filePath = filePath, ext = ext)
     
     def ConvertVideoToWavRPC(self, request, context):
         filePath, ext = ffmpeg.convertVideoToWav(request.filePath)
         return ct_pb2.File(filePath = filePath, ext = ext)
 
+    def ConvertVideoToWavRPCWithOffset(self, request, context):
```

Comment:
Lets remove the pre-existing, ConvertVideoToWavRPC, and in your function add a check to see if Offset does not have a value, assume it to be 00:00